tuf/formats.py

```diff
@@ -151,6 +151,9 @@
 NAME_SCHEMA = SCHEMA.AnyString()
 NAMES_SCHEMA = SCHEMA.ListOf(NAME_SCHEMA)
 
+# A string representing data.
+DATA_SCHEMA = SCHEMA.AnyString()
```

Comment:
I wonder if that will do what it should, with the Python2/3 `str`/ `bytes` dichotomy.